src/index.html

```diff
@@ -12,7 +26,50 @@
       href="./styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="/src/images/imac.jpeg"
+        alt="imac photo"
```

Comment:
``alt`` attribute should describe the image content

![image](https://github.com/mate-academy/layout_product-cards/assets/54860641/18e02c82-3b69-4aee-9c5a-5ef13b150a6e)